### PR TITLE
Create a big ole' text banner for env

### PR DESCRIPTION
**Pull request by jdotpz**
Since the same job is firing off both staging and prod, it seemed wise to splatter the logs with a big old readable env.

(Ass seen here (https://leeroy.mocotoolsstaging.net/job/socorro-master/143/console) and here(https://leeroy.mocotoolsstaging.net/job/socorro-master/142/console)

### CODE REVIEW

**Comment by phrawzty** [comment]
Looks like you accidentally pulled in too many commits (dbf6b79 and 80385bd at a minimum, but everything before c090f0b is suspect).

File: bin/deploy-socorro.sh [hunk 1]
```diff
@@ -65,6 +65,22 @@ function error_check() {
 ## PROGRAM FUNCTIONS
 function format_logs() {
```
Comment (phrawzty) [inline bin/deploy-socorro.sh]: Recommend changing the function name to `environment_banner()` or something - `format_logs()` suggests that it's, uh, formatting logs. :blush:

**Comment by phrawzty** [comment]
Fun idea! You'll need to massage those echoes a bit in order to make sure they output properly (right now the lines aren't aligned).  You may wish to enclose the text in single quotes in order to prevent shell interpolation - this way you won't run into issues with parentheses, pipes, and so forth.

Example:
```bash
function environment_banner() {
    if [ "$ENVNAME" == "stage" ]; then
        echo ' _____ _'
        echo '/  ___| |'
        echo '\ `--.| |_ __ _  __ _  ___'
        echo ' `--. \ __/ _` |/ _` |/ _ \ '
        echo '/\__/ / || (_| | (_| |  __/'
        echo '\____/ \__\__,_|\__, |\___|'
        echo '                 __/ |'
        echo '                |___/'
    fi
    if [ "$ENVNAME" == "prod" ]; then
        echo '______              _'
        echo '| ___ \            | |'
        echo '| |_/ / __ ___   __| |'
        echo '|  __/ `__/ _ \ / _` |'
        echo '| |  | | | (_) | (_| |'
        echo '\_|  |_|  \___/ \__,_|'
    fi
}
```

**Comment by jdotpz** [comment]
ah, I'll redo the PR and not pull in other commits.
Good call on the single quotes, your ascii is far more readable, and readable > fun fonts. Thanks :D

**Pull request closed without merging**